.travis.yml

```diff
@@ -10,3 +10,15 @@ notifications:
     on_success: change  # options: [always|never|change] default: always
     on_failure: always  # options: [always|never|change] default: always
     on_start: never     # options: [always|never|change] default: always
+
+deploy:
+  provider: releases
+  api_key: "GITHUB OAUTH TOKEN"
```

Comment:
Your user settings (top right) / personal access tokens / new. Then give it 'repo' access.